### DISCUSSION TITLE
Hm-IP-Mod-Ho keine Steuerung mehr möglich

### DISCUSSION THREAD

**Discussion by micklafisch** [OpenCCU/OpenCCU#2658] [🇩🇪 Fragen&Antworten]
Hallo,
eigentlich wollte ich nur ein URL Aufruf zum ansteuern des Homematic Hörmann Moduls erstellen, da ist mir aufgefallen dass das Gerät immer wieder auf "Gerätekommunikation gestört" gestellt wird. Wenn ich mit der Originalfernbedienung das Hörmann-Tor verfahre, dann ist die Kommunikationsstörung weg und ich stehe den korrekten Status in der CCU3. Wenn ich dort aber egal auf welchen Button auch immer gehe, kommt die Störung sofort wieder. Meine iobroker vis welche auf den State "HmIP-RF.0024xxxxxxxxxx:1.DOOR_COMMAND" losgeht funktioniert auch nicht mehr. 

Bis auf die Updates der CCU3 (aktuell auf 3.73.9.20240130) gabs keine Änderungen - leider kann ich nicht sagen mit welcher Version sich der Fehler eingeschlichen hat. 

Wie kann ich das weiter eingrenzen bzw. was muss ich tun um den Fehler zu eleminieren?

Grüße

**Comment by 42Joe** [comment]
Hallo micklafisch,
oje, keine Rückmeldung. Ich hab dasselbe Problem. Gibt's schon eine Lösung dafür?
LG

**Comment by micklafisch** [comment]
Hi, ich hatte meine CCU3 mit einer externen Stabantenne ausgestattet, damit war das Problem weg. Scheinbar war der Empfang gerade so an der Grenze zum kippen

**Reply by 42Joe** [reply]
Danke, das versuche ich mal.
